src/EmployerRequestApprenticeTraining/SFA.DAS.EmployerRequestApprenticeTraining.MockApis/Program.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using WireMock.Server;
+
+namespace SFA.DAS.EmployerRequestApprenticeTraining.MockApis
+{
+    [ExcludeFromCodeCoverage]
+    public static class Program
```

Comment:
@ChrisJWoodcock silly question what does the mock api do / used for ?

Reply:
Nothing currently but when this is talking to other inner api's it can useful to mock them